package.json

```diff
@@ -30,20 +30,23 @@
     "url": "https://github.com/wikimedia/swagger-router/issues"
   },
   "dependencies": {
-    "bluebird": "^3.4.0",
-    "js-yaml": "^3.6.1",
+    "bluebird": "^3.5.2",
+    "js-yaml": "^3.12.0",
     "tassembly": "^0.2.3",
-    "template-expression-compiler": "^0.1.8",
-    "cassandra-uuid": "^0.0.2"
+    "template-expression-compiler": "^0.1.10",
+    "cassandra-uuid": "^0.1.0"
   },
   "devDependencies": {
-    "mocha": "^3.1.0",
+    "mocha": "^5.2.0",
     "mocha-jshint": "^2.3.1",
```

Comment:
Don't need to depend on it anymore.

Reply:
They are peer dependencies of `eslint-config-node-services` and without both eslint fails

Reply:
I'm pretty sure that is not true https://github.com/wikimedia/eslint-config-node-services/blob/master/package.json#L26-L29

Reply:
sorry I was thinking of `eslint-plugin-jsdoc`